src/libs/Navigation/AppNavigator/Navigators/FullScreenNavigator.tsx

```diff
@@ -19,20 +20,22 @@ function FullScreenNavigator() {
     const screenOptions = getRootNavigatorScreenOptions(isSmallScreenWidth, styles, StyleUtils);
 
     return (
-        <View style={styles.rootNavigatorContainerStyles(isSmallScreenWidth)}>
-            <RootStack.Navigator>
-                <RootStack.Screen
-                    name={SCREENS.WORKSPACE.INITIAL}
-                    options={screenOptions.homeScreen}
-                    getComponent={loadWorkspaceInitialPage}
-                />
-                <RootStack.Screen
-                    name={SCREENS.WORKSPACES_CENTRAL_PANE}
-                    options={screenOptions.centralPaneNavigator}
-                    component={ModalStackNavigators.WorkspaceSettingsModalStackNavigator}
-                />
-            </RootStack.Navigator>
-        </View>
+        <FocusTrapForScreens>
```

Comment:
@adamgrzybowski Why do we have `FocusTrapForScreens` here if each child screen already has a screenwrapper containing it?

Reply:
@wildan-m hello, I also worked on this PR so I can answer. The reason is that `FullScreenNavigator` can display two screens, for example: 
![image](https://github.com/user-attachments/assets/e5422947-dba8-4317-af2f-119404d8073f)
In this case we have to disable `FocusTrap` which is rendered by `ScreenWrapper`. Otherwise we would have trap active only in red screen or blue screen. `FocusTrap` over navigator allows us to trap focus in both visible screens.
You can see screens with disabled focus trap in `src/components/FocusTrap/WIDE_LAYOUT_INACTIVE_SCREENS.ts` file.

Reply:
@jnowakow Thank you for your response. Although I haven't encountered any actual issues yet, I noticed that on smaller screens, when the trap is disabled in the screen wrapper (for some reason), it still traps the focus because the parent container FullScreenNavigator's focus trap exists.

Reply:
Okey, so probably it's desired behaviour